addons/website_livechat/models/discuss_channel_member.py

```diff
@@ -0,0 +1,22 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models
+
+
+class DiscussChannelMember(models.Model):
+    _inherit = "discuss.channel.member"
+
+    def _write_after_hook(self, vals):
```

Comment:
Note: must be after write to avoid having to check channel/member existence inside write